config/secrets/addon-params-secret.yaml

```diff
@@ -30,4 +33,10 @@ parameters:
   - name: PORT
     value: "567"
   - name: FROM
-    value: "test@example.com"
\ No newline at end of file
+    value: "test@example.com"
+  - name: STS_ROLE_ARN
+    value: "arn:aws:iam::111111111111:role/11111"
```

Comment:
This is a dummy value which satisfies the format for an AWS role ARN.
This value is being set in the `make cluster/prepare/addon-params`, which is being used in `make cluster/prepare` and `make cluster/prepare/local`. Meaning that if we ran `make cluster/prepare/local` for a non-sts cluster, it will still set its value to a dummy value, which may not be favourable. I can add an if statement here to determine that 👍🏻